website/versioned_docs/version-1.0.0-beta2/button.md

```diff
@@ -0,0 +1,80 @@
+---
+id: version-1.0.0-beta2-button
+title: Button
+original_id: button
+---
+
+Buttons can be used to interact with the screen. It takes the following props.
+
+<img src="https://i.imgur.com/ptrwAVW.png" width="300" />
+
+```js
+import { Button } from 'react-native-elements';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+<Button
+  text='BUTTON'
+/>
+
+<Button
+  icon={
+    <Icon
+      name='arrow-right'
+      size={15}
+      color='white'
+    />
+  }
+  text='BUTTON WITH ICON'
+/>
+
+<Button
+  icon={
+    <Icon
+      name='arrow-right'
+      size={15}
+      color='white'
+    />
+  }
+  iconRight
+  text='BUTTON WITH RIGHT ICON'
+/>
+
+<Button
+  text="LOADING BUTTON"
+  loading
+  loadingProps={{ size: "large", color: "rgba(111, 202, 186, 1)" }}
+  textStyle={{ fontWeight: "700" }}
+  buttonStyle={{
+    backgroundColor: "rgba(92, 99,216, 1)",
+    width: 300,
+    height: 45,
+    borderColor: "transparent",
+    borderWidth: 0,
+    borderRadius: 5
+  }}
+  containerStyle={{ marginTop: 20 }}
+/>
+```
+
+#### Button props
+
+> Also receives all TouchableNativeFeedback (Android) or TouchableOpacity (iOS) props
+
+| prop | default | type | description |
+| ---- | ---- | ----| ---- |
+| text | none | string | button text (optional) |
+| textStyle | none | Text style (object) | add additional styling for text component (optional) |
+| textProps | none | object (style) | add additional props for Text component (optional) |
```

Comment:
Could you just make these changes I made on `next` ?
`text` -> `title`
`textStyle` -> `titleStyle`
`textProps` -> `titleProps`

Reply:
Sure

Reply:
but I think they're for beta3 right?

Reply:
True !

Reply:
Done!